problems/isOdd.js

```diff
@@ -7,7 +7,12 @@
 
 */
 
-function isOdd() {
+function isOdd(n) {
+    if (n % 2 === 1){
```

Comment:
If you used an implicit return, would you still need an if-else statement? Try condensing this code to one line.